drivers/hid/hidparse/hidp_km.c

```diff
@@ -0,0 +1,254 @@
+/*
+ * PROJECT:     ReactOS HID Parser Library
+ * LICENSE:     GPL-3.0-or-later (https://spdx.org/licenses/GPL-3.0-or-later)
+ * FILE:        drivers/hid/hidparse/hidp_km.c
+ * PURPOSE:     HID Parser kernel mode
+ * COPYRIGHT:   Copyright (C) Michael Martin (michael.martin@reactos.org)
+ *              Copyright (C) Johannes Anderwald (johannes.anderwald@reactos.org)
+ *              Copyright (C) Roman Masanin (36927roma@gmail.com) 2022
+ */
```

Comment:
Please use the new file header as follows: https://reactos.org/wiki/Coding_Style#File_Structure